reference/configuration/framework.rst

```diff
@@ -255,8 +255,9 @@ Configuration
   * :ref:`name <reference-workflows-name>`
 
     * `audit_trail`_
-    * `initial_place`_
```

Comment:
@javiereguiluz is it ok to remove the deprecated one here? Or do we need to leave some info about the before option name?